src/components/Catalog.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from "react";
+import Movie from "./Movie";
+import { FaFilter } from "react-icons/fa";
+
+class Catalog extends Component {
+  constructor() {
+    super();
+    this.state = {
+      value: ""
+    };
+  }
+  handleChange = event => {
+    this.setState({ value: event.target.value });
+  };
+  render() {
+    const user = this.props.users[this.props.match.match.params.id];
+    let rented = user.movies.filter(m => m.isRented);
+    let unRented = user.movies.filter(m => !m.isRented);
+    if (this.state.value.length > 0) {
+      const releventMovies = user.movies.filter(m =>
+        m.title.toLowerCase().includes(this.state.value)
+      );
+      rented = releventMovies.filter(m => m.isRented);
+      unRented = releventMovies.filter(m => !m.isRented);
+    }
```

Comment:
Use `if else` statement here.
Somthing like:
` 
if (this.state.value.length > 0) {
      const releventMovies = user.movies.filter(m =>
        m.title.toLowerCase().includes(this.state.value)
      );
      rented = releventMovies.filter(m => m.isRented);
      unRented = releventMovies.filter(m => !m.isRented);
    } else {
rented = user.movies.filter(m => m.isRented);
    unRented = user.movies.filter(m => !m.isRented);
}`
if the condtion is true there is no need to filter the movies before the `if`